PSKoans/Koans/Foundations/AboutStrings.Koans.ps1

```diff
@@ -102,7 +102,7 @@ Describe 'Strings' {
 
         It 'adds strings together' {
             # Two become one.
-            $String1 = 'This string'
+            $String1 = '__'
```

Comment:
```suggestion
            $String1 = '_____'
```

Been trying to move towards using long-blanks for strings as well, just to make it more clear that we're expecting more than a couple of characters. 🙂